cmd/guacone/cmd/vulnerability.go

```diff
@@ -329,11 +332,16 @@ func searchDependencyPackages(ctx context.Context, gqlclient graphql.Client, top
 	return path, tableRows, nil
 }
 
-func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilityNodeID string, edgeType model.Edge, depth, pathsToReturn int) ([]string, error) {
+func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilityNodeID string, edgeType model.Edge, depth, pathsToReturn int) ([]string, []table.Row, error) {
 	var path []string
+	var tableRows []table.Row
 	vulnNodeNeighborResponse, err := model.Neighbors(ctx, gqlclient, vulnerabilityNodeID, []model.Edge{edgeType})
 	if err != nil {
-		return nil, fmt.Errorf("error querying neighbor for vulnerability: %w", err)
+		return nil, []table.Row{}, fmt.Errorf("error querying neighbor for vulnerability: %w", err)
+	}
+	pkgVersionNeighborResponse, err := model.Neighbors(ctx, gqlclient, topPkgResponse.Packages[0].Namespaces[0].Names[0].Versions[0].Id, []model.Edge{model.EdgePackageCertifyVexStatement})
```

Comment:
This was a hack I did to unblock myself, but would like feedback about what we should actually do here

Reply:
no need for this, see the comment above. This can be done from neighbors from the vulnerability